common/djangoapps/student/views.py

```diff
@@ -894,6 +894,23 @@ def dashboard(request):
     return response
 
 
+def get_refund_status(request):
+    # get course key
+    course_key = request.GET.get('course_id')
+    if not course_key:
+        course_enrollment = CourseEnrollment.get_enrollment(request.user, course_key)
+
+        user_already_has_certs_for = GeneratedCertificate.course_ids_with_certs_for_user(request.user)
+        refundable_status = True if course_enrollment.refundable(
```

Comment:
Im not comfortable with this . we should handle this properly. what if the otto call fails . we will have to show appropriate message on our end and disable the unenroll button . this will cause negative user experience. @schenedx  thoughts.